tests/integration/components/donation/credit-card-test.js

```diff
@@ -50,6 +54,4 @@ test('it sends submit with credit card fields when button is clicked', function(
       .cardYear(expectedProps.year)
       .cardCVC(expectedProps.cvc)
       .clickSubmit();
-
-  assert.ok(page.submitDisabled, 'Submit button got disabled.');
```

Comment:
The component is destroyed at this point, so the `_afterSubmit` method returns early and does not disable the button.